.wrgl/config.yaml

```diff
@@ -1848,6 +1848,18 @@ branch:
     file: data/fuse/per_rayne_pd.csv
     primaryKey:
       - uid
+  uof-citizens-new-orleans-pd:
+    remote: uof-citizens-new-orleans-pd
+    merge: refs/heads/main
+    file: data/fuse/uof_citizens_new_orleans_pd.csv
+    primaryKey:
+      - uof_citizen_uid
+  uof-officers-new-orleans-pd:
+    remote: uof-officers-new-orleans-pd
+    merge: refs/heads/main
+    file: data/fuse/uof_officers_new_orleans_pd.csv
+    primaryKey:
+      - uid
```

Comment:
Please add entries for `uof-citizen` and `uof-officer`.

Reply:
Thanks. Just pushed the changes.